### PR TITLE
Added Process For Creating New SIG (document).

### PR DESCRIPTION
**Pull request by xsub**
A draft for AlmaLinux Foundation SIG Lifecycle Guidelines.

### CODE REVIEW

**Review by bennyvasquez — COMMENTED** [review]
An incredible start! Lots of nit-picking about formatting and making sure we're calling the foundation by the correct name (AlmaLinux OS Foundation). Some things also require adjusting the text. @xsub once you take a look at my suggestions and required edits, we'll get some wider input here.

File: docs/sigs/ProcessForCreatingNewSIG.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+To ensure that SIGs continue to provide value and effectively contribute to the AlmaLinux ecosystem, introduce a periodic evaluation process. This process may involve:
+
+a. Reviewing the progress and achievements of each SIG against their stated objectives and timelines.
+
+b. Gathering feedback from SIG members and the broader AlmaLinux community on the performance and impact of the SIG.
+
+c. Identifying areas for improvement and implementing changes to enhance the SIG's effectiveness and alignment with the community's goals.
+
+
+###
```
[the remark this answers is not in the document]
Comment (xsub) [reply docs/sigs/ProcessForCreatingNewSIG.md:99]: I am sure we can discuss with @jaboutboul.

File: docs/sigs/ProcessForCreatingNewSIG.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+### A.3. SIG Decommissioning
+
+In the event a SIG becomes inactive, no longer serves its intended purpose, or does not align with the community's objectives, outline a process for decommissioning the SIG. This may include:
+
+a. Notifying the AlmaLinux Foundation Board of the intention to decommission the SIG.
+
+b. Ensuring that all relevant documentation, resources, and knowledge are preserved and transferred to appropriate parties within the community.
+
+c. Formally announcing the decommissioning of the SIG and providing a rationale for the decision.
```
[the remark this answers is not in the document]
Comment (xsub) [reply docs/sigs/ProcessForCreatingNewSIG.md:110]: Accepting as is, will refine later.

**Comment by xsub** [comment]
All good, please merge.

**Review by xsub — COMMENTED** [review]
ACK.

**Review by bennyvasquez — APPROVED** [review]

**Comment by bennyvasquez** [comment]
LGTM

**Comment by bennyvasquez** [comment]
LGTM!

**Pull request merged**